frontend/packages/dev-console/src/components/project-access/__tests__/ProjectAccess.spec.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { Formik } from 'formik';
+import { LoadingBox, StatusBox } from '@console/internal/components/utils';
+import ProjectAccess from '../ProjectAccess';
+
+type ProjectAccessProps = React.ComponentProps<typeof ProjectAccess>;
+
+describe('Project Access', () => {
+  it('should show the LoadingBox when role bindings are not loaded, but user has access to role bindings', () => {
+    const projectAccessProps: ProjectAccessProps = {
```

Comment:
You can declare this inside beforeEach block and change the specific prop of this object in each test case. No need to declare this in each test case